<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ValueNoiseStarter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ValueNoise</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择你的算法：</string>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Help">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wwdx">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>未完待续</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PerlinNoiseStarter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PerlinNoise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MidPointStarter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MidPoint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ErosionStarter">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Erosion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Settings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <zorder>quickWidget</zorder>
   <zorder>label</zorder>
   <zorder>ValueNoiseStarter</zorder>
   <zorder>Help</zorder>
   <zorder>wwdx</zorder>
   <zorder>PerlinNoiseStarter</zorder>
   <zorder>MidPointStarter</zorder>
   <zorder>ErosionStarter</zorder>
   <zorder>Settings</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
